app/assets/javascripts/components/notes/Note.js

```diff
@@ -1,49 +1,35 @@
 import React from 'react';
 
-class Note extends React.Component {
-  constructor(props) {
-    super(props);
-    this._handleDelete = this._handleDelete.bind(this);
-  }
+const Note = ({ note, disabled, handleDelete }) => {
 
-  _handleDelete() {
-    const { note, handleDelete } = this.props;
-    handleDelete(note);
+  const _handleDelete = () => {
+    handleDelete(note)
   }
 
-  parseNote(note) {
+  const parseNote = () => {
     return ({
       __html: window.md.makeHtml(note.escape('note'))
     });
   }
```

Comment:
What do you think about using `useMemo` (https://beta.reactjs.org/apis/react/useMemo#usememo) here to avoid re-parsing something that hasn't changed? The value used in the dependencies array could be `note.note` in a way that we would only call `note.escape('note')` and `makeHtml` inside the calculation function.